packages/asc-ui/src/components/Select/Select.tsx

```diff
@@ -82,16 +80,6 @@ const Select = React.forwardRef<
       }
     }, [ref])
 
-    useEffect(() => {
-      if (inputValue && ref.current) {
-        setValue(inputValue)
-
-        // The selectedValue label doesn't update properly if you send a new value prop to this component and the component doesn't rerender
-        ref.current.value = inputValue
-        ref.current.dispatchEvent(createEvent('change', { bubbles: true }))
-      }
-    }, [inputValue, ref])
-
```

Comment:
Now that this code has been removed are we not introducing the original bug it attempted to fix? If possible we should add a test that changes the `value` prop to see if the overlaying label gets updated properly.

Reply:
I've tested the behaviour in several browsers and it all works as expected. I'll add a test that will cover the display value of the overlaying label.